filters/scanner.go

```diff
@@ -249,7 +249,7 @@ func digitVal(ch rune) int {
 }
 
 func isFieldRune(r rune) bool {
-	return (r == '_' || isAlphaRune(r) || isDigitRune(r))
+	return (r == '-' || r == '_' || isAlphaRune(r) || isDigitRune(r))
```

Comment:
The change looks good. Can you add some tests in https://github.com/containerd/containerd/blob/main/filters/scanner_test.go#L38?